src/main/java/net/minecraftforge/event/ServerParticleSpawnEvent.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * Minecraft Forge
+ * Copyright (c) 2016-2019.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation version 2.1
+ * of the License.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+package net.minecraftforge.event;
+
+import net.minecraft.particles.IParticleData;
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.eventbus.api.Cancelable;
+import net.minecraft.network.play.server.SSpawnParticlePacket;
+import net.minecraftforge.eventbus.api.Event;
+
+/**
+ * ParticleWorldHandleEvent is fired before an {@link SSpawnParticlePacket} is sent.<br>
+ * <br>
+ * This event is {@link Cancelable}. <br>
+ * If the event is canceled, the execution of the command does not occur.<br>
```

Comment:
Copy-paste error?

Reply:
Yep sorry about that, I've changed it now.